datasets/us/cbp_forced_labor/us_cbp_forced_labor.yaml

```
@@ -0,0 +1,354 @@
+              - Sichuan (Szechuan) Pin Chiang Enterprise Company
+      - match: 'Sichuan (Szechuan) Xinsheng (New Life) Labor Factory, a/k/a Xinsheng (New Life) Labor Factory'
+        entities: 
+          - name:  
+              - Sichuan (Szechuan) Xinsheng (New Life) Labor Factory
+            alias:
+              - Xinsheng (New Life) Labor Factory
+      - match: 'Da Wei Chemical Factory'
+        entities: 
+          - name: 
+              - Da Wei Chemical Factory
+      - match: 'Zi Gong Machinery Factory, a/k/a Zigong Machinery Factory, a/k/a Sichuan (Szechuan) Zigong Labor Reform Detachment'
+        entities: 
+          - name: 
+              - Zi Gong Machinery Factory
+            alias:
+              - Zigong Machinery Factory
+              - Sichuan (Szechuan) Zigong Labor Reform Detachment
+      - match: 'Ya An Auto Parts Factory, a/k/a Sichuan (Szechuan) Bin-Jiang Enterprises Company'
+        entities: 
+          - name:  
+              - Ya An Auto Parts Factory
+            alias:
+              - Sichuan (Szechuan) Bin-Jiang Enterprises Company
+      - match: 'Miao Chi Tea Farm'
+        entities: 
+          - name: 
+              - Miao Chi Tea Farm
+      - match: 'Wang Tsang Coal and Iron Factory'
+        entities: 
+          - name: 
+              - Wang Tsang Coal and Iron Factory
+      - match: 'Qinghai Hide & Garment Factory, a/k/a Qinghai Leather and Wool Bedding and Garment Factory, a/k/a Qinghai Fur and Cloth Factory'
+        entities: 
+          - name:  
+              - Qinghai Hide & Garment Factory
+            alias:
+              - Qinghai Leather and Wool Bedding and Garment Factory
+              - Qinghai Fur and Cloth Factory
+      - match: 'Shandong Laiyang Heavy Duty Machinery Factory'
+        entities: 
+          - name: 
+              - Shandong Laiyang Heavy Duty Machinery Factory
+      - match: 'Xuzhou Forging and Pressing Machine Works'
+        entities: 
+          - name: 
+              - Xuzhou Forging and Pressing Machine Works
+      - match: 'Yunnan Machinery, a/k/a Golden Horse (JinMa) Diesel Factory, a/k/a Yunnan 1st Prison'
+        entities: 
+          - name: 
+              - Yunnan Machinery
+            alias:
+              - Golden Horse (JinMa) Diesel Factory
+              - Yunnan 1st Prison
+      - match: 'Xiang-Yang Machinery Plant'
+        entities: 
+          - name: 
+              - Xiang-Yang Machinery Plant
+      - match: 'Artisanal Small Mines'
+        entities: 
+          - name: 
+              - Artisanal Small Mines
+      - match: 'Central Romana Corporation Limited'
+        entities: 
+          - name: 
+              - Central Romana Corporation Limited
+      - match: 'Mangalore Ganesh Beedie Works'
+        entities: 
+          - name: 
+              - Mangalore Ganesh Beedie Works
+      - match: 'Fuchu Prison; Union Kogyo Co., Ltd.'
+        entities: 
+          - name: 
+              - Fuchu Prison
```

If there's generally a single value for name, you can just make the name the value - no point putting it in an array. This will save a lot of visual space reading the lookups

If there is a case where you want two name values, putting it in an array is fine. Should just work.

```suggestion
          - name: Fuchu Prison
```